### DISCUSSION TITLE
Reconnecting with the previously paired network.

### DISCUSSION THREAD

**Discussion by mithunGhosh13** [openthread/openthread#7626] [Q&A] [answered]
Hello,
I am using nrf52833 for my project and i have established a thread network to communicate with a nrf52840 device . I am able to paired both the devices . I can see device got disconnected from the network while power goes off , it is fine. But i wanted to rejoin into the previously connected network when device is powered again with out doing any physical activities with the device . Is there any possibilities to rejoin into thread network if so please let me know.

Thank you.

**Comment by jwhui** [comment, marked as answer]
This appears to be a duplicate of https://github.com/openthread/openthread/discussions/7607

See answer provided in https://github.com/openthread/openthread/discussions/7607#discussioncomment-2595144

**Reply by mithunGhosh13** [reply]
yes .but i am not able to connect devices. 
if once we paired, is it needs to be pair again while joining into the network?
Very first time i am pairing the devices by pressing the key but i don't wanted to press key while device tries to rejoin into same network after got disconnect once. Device should join automatically. 
How it would be possible?

**Reply by jwhui** [reply]
As mentioned in https://github.com/openthread/openthread/discussions/7607#discussioncomment-2610353:

Given that this appears to be an issue with Nordic's example code, please reach out via [Nordic DevZone](https://devzone.nordicsemi.com/) for support.

Please do not create a new discussion for the same topic.
